nx-X11/programs/Xserver/hw/nxagent/Args.c

```diff
@@ -283,7 +283,16 @@ int ddxProcessArgument(int argc, char *argv[], int i)
 
     if (nxagentOptionFile)
     {
-      nxagentProcessOptionsFile();
+      /* if the "filename" starts with nx/nx treat it
+	 as an option string */
+      if (strncasecmp(nxagentOptionFile, "nx/nx", 5) == 0)
+      {
+	nxagentParseOptionString(nxagentOptionFile + 5);
+      }
+      else
+      {
+	nxagentProcessOptionsFile(nxagentOptionFile);
+      }
```

Comment:
About the commit msg of this commit: it does actually fix #476, right? Why do you use "addresses"?